packages/ckeditor5-media-embed/docs/features/media-embed.md

```diff
@@ -101,6 +101,16 @@ By default, the media embed feature outputs semantic `<oembed url="...">` tags f
 </figure>
 ```
 
+Further customization of semantic data output can be done through the {@link module:media-embed/mediaembed~MediaEmbedConfig#elementName `config.mediaEmbed.elementName`}. As an example, if `elementName` is set to `o-embed`:
+
+```html
+<figure class="media">
+	<o-embed url="https://media-url"></o-embed>
+</figure>
+```
+
+To be backward compatible with legacy semantic elements, `<oembed>` tags are still supported when `elementName` is set.
```

Comment:
I don't understand this sentence. The `<oembed>` will be used when `elementName` is set to what?

`module:media-embed/mediaembed~MediaEmbedConfig#elementName` suggests that `<oembed>` is used when `elementName` is unset, so I think the above contradicts the API docs.

Reply:
Even if `elementName` is overridden, `oembed` will [still be upcasted](https://github.com/ckeditor/ckeditor5/pull/9418/files#diff-448f814f228e15415ca94cb35807f9ac08f017ab3761d3d184fa7554f41945d7R225) for backward compatibility purposes.

I'll rewrite the sentence to see if it makes more sense :)